guillotina/api/service.py

```diff
@@ -36,6 +42,30 @@ def __setattr__(self, name, value):
 
 
 class Service(View):
+    __validator__ = None
+
+    async def validate(self):
+        data = await self.request.json()
+        if(self.__class__.__validator__ is None):
+            try:
+                self.__class__.__schema__ = self.__config__['parameters'][0]['schema']['$ref'].split('/')[-1]
+                self.__class__.__validator__ = get_schema_validator(self.__class__.__schema__)
+            except Exception as e:
+                logger.warning('Invalid jsonschema: {}'.format(e))
```

Comment:
Can we just iterate over the parameters and inspect the config to get the values correct instead of having a blind except block here?